# Generated by Django 4.2.16 on 2024-12-31 08:56

import datetime
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('reputation_app', '0093_siteanalytics_error_siteanalytics_is_error_and_more'),
    ]

    operations = [
        migrations.AlterField(
            model_name='company',
            name='company_free_trial_expiry',
            field=models.DateTimeField(default=datetime.datetime(2025, 1, 5, 8, 56, 37, 681653, tzinfo=datetime.timezone.utc)),
        ),
        migrations.AlterField(
            model_name='company',
            name='company_subscription_date',
            field=models.DateTimeField(default=datetime.datetime(2024, 12, 31, 8, 56, 37, 681653, tzinfo=datetime.timezone.utc)),
        ),
        migrations.AlterField(
            model_name='companybotchats',
            name='date_sent',
            field=models.DateTimeField(default=datetime.datetime(2024, 12, 31, 8, 56, 37, 681653, tzinfo=datetime.timezone.utc)),
        ),
        migrations.AlterField(
            model_name='companyfacebook',
            name='token_expiry',
            field=models.DateField(default=datetime.datetime(2025, 3, 1, 8, 56, 37, 687655, tzinfo=datetime.timezone.utc)),
        ),
        migrations.AlterField(
            model_name='companyinstagram',
            name='token_expiry',
            field=models.DateField(default=datetime.datetime(2025, 3, 1, 8, 56, 37, 687655, tzinfo=datetime.timezone.utc)),
        ),
        migrations.AlterField(
            model_name='companytiktok',
            name='token_expiry',
            field=models.DateField(default=datetime.datetime(2025, 3, 1, 8, 56, 37, 688655, tzinfo=datetime.timezone.utc)),
        ),
        migrations.AlterField(
            model_name='siteanalytics',
            name='date_visited',
            field=models.DateTimeField(default=datetime.datetime(2024, 12, 31, 8, 56, 37, 680653, tzinfo=datetime.timezone.utc)),
        ),
    ]
